instrumentation/spring/spring-rmi-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/rmi/v4_0/server/ServerInstrumentation.java

```diff
@@ -41,49 +41,50 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class InvokeMethodAdvice {
 
+    public static class AdviceLocals {
```

Comment:
I think you forgot to refactor this one to the `AdviceScope` pattern? Or was it intentional?